### PR TITLE
Create-desktop-version

### PR DESCRIPTION
**Pull request by Abdelaziz-Mahdi**
**The Fourth Project: Desktop Version**

- Use media queries to make sure that the website displays correctly on desktop screens.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -559,3 +597,612 @@ label {
...
+  #item10 .infoBox {
+    display: none;
+  }
+
+  #itemFlow {
+    position: absolute;
+    right: 0;
+    top: 620px;
+    display: grid;
+  }
```
Comment (pinkmoon25) [inline styles.css]: - [ ] Your project is matchng the figma design very well, only the second project is not aligned properly, it should share the first one's row but is bigger in height your design :point_down: 
image

figma design :point_down: 
image

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @Abdelaziz-Ali , 

Good job so far! you have done really well implementing the dektop version :clap: :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
linter checks :heavy_check_mark: 
professional readme :heavy_check_mark: 
follows figma template :heavy_check_mark: 
used flex and grid :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (Abdelaziz-Mahdi) [reply styles.css]: I have corrected the size of the item, if it`s still wrong please give a code hint as I don`t know more than what I already did, it looks correct when the browser width is 1440 as Figma width.

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED  
 ## Hi  @Abdelaziz-Ali 
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Your repo is professional :white_check_mark:
 * Well structured files :white_check_mark:
 * No Linter errors :white_check_mark:
 * use new readme template provided by microverse :white_check_mark:
 
 Now you can merge this branch and move on :1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers four_leaf_clover:

 
 
 ## Optional suggestions
- [ ]  N/A

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**